### ISSUE TITLE
Ansible-lint does not catch empty tags

### ISSUE THREAD

**Issue by mrsiesta** [willthames/ansible-lint#180]
# Issue Type
- Bug report

# Ansible and Ansible Lint details

```
ansible 2.1.0.0
ansible-lint 3.0.0rc5
```

- ansible installation method: installed via apt, deb: 2.1.0.0-1ppa~trusty
- ansible-lint installation method: pip

# Desired Behaviour

In ansible 2.1.0.0 when there is an empty tags: section, a linting error should be raised.

# Actual Behaviour (Bug report only)
No warning is raised in the linter when an empty - tags: section is defined.

In ansible 2.1.0.0 if you have an empty `-tags: ` section in your playbook, ansible will throw with the following error:
`ERROR! tags must be specified as a list`

example playbook:
```
 - name: Install Docker
   apt: pkg=docker-engine state=present
-  tags:
```

**Comment by willthames** [comment]
ansible-lint isn't designed to catch problems caught by ansible-playbook --syntax-check.

The issue template even asks for the results of running under the syntax check.
